### PR TITLE
Remove manifest plugin

### PR DESCRIPTION
**Pull request by timkelty**
As discussed here: https://github.com/neutrinojs/neutrino/pull/1253#issuecomment-447009554

### CODE REVIEW

File: packages/react-components/index.js [hunk 1]
```diff
@@ -10,7 +10,6 @@ module.exports = (neutrino, opts = {}) => {
     html: process.env.NODE_ENV === 'development' && {
       title: 'React Preview'
     },
-    manifest: process.env.NODE_ENV === 'development',
     externals: opts.externals !== false && {},
```
Comment (timkelty) [inline packages/react-components/index.js]: @edmorley any idea what purpose this was serving, if any?

File: packages/web/index.js [hunk 2]
```diff
@@ -38,7 +38,6 @@ module.exports = (neutrino, opts = {}) => {
       hot: opts.hot !== false
     },
     style: {},
-    manifest: opts.html === false ? {} : false,
```
Comment (edmorley) [inline packages/web/index.js]: Should we now throw if a `manifest` option is passed? (Like occurs for the other removed options in this file)

Comment (timkelty) [reply packages/web/index.js]: 👍

File: packages/react-components/index.js (hunk 1 above)
Comment (edmorley) [reply packages/react-components/index.js]: I have no idea :-/

@eliperelman / @helfi92 do you?

File: packages/web/index.js [hunk 3]
```diff
@@ -82,6 +81,13 @@ module.exports = (neutrino, opts = {}) => {
     );
   }
 
+  if ('manifest' in options) {
+    throw new ConfigurationError(
```
Comment (edmorley) [inline packages/web/index.js]: I don't suppose you could add a test for this? Similar to:
https://github.com/neutrinojs/neutrino/blob/21225bfae3d83ca6e687858f246d8a4f883399d9/packages/web/test/web_test.js#L257-L311

Comment (timkelty) [reply packages/web/index.js]: @edmorley added test!

**Pull request merged**